### PR TITLE
[SP-3767] Backport of CDE-912 - Uncaught Error: Load timeout for modules when embedding dashboard within Pentaho Server (7.1 Suite)

### PR DESCRIPTION
**Pull request by Advent51**
- modified backport of #1145

### CODE REVIEW

**Comment by Advent51** [comment]
@pamval @mchen-len-son @davidmsantos90 @pentaho/millenniumfalcon please review

**Comment by dcleao** [comment]
@davidmsantos90 I think this one is for you to review.

File: impl/client/src/main/javascript/web/pentaho/i18n.js [hunk 1]
```diff
@@ -104,11 +107,14 @@ define([
...
-    // Remove basePath from bundle url
     var serverUrl = context.server.url;
-    var basePath = (serverUrl && serverUrl.pathname) || "";
-    if(basePath && absBundleUrl.indexOf(basePath) === 0)
-      absBundleUrl = absBundleUrl.substr(basePath.length);
+
+    // Remove basePath from bundle url
+    // Taking into account embedded scenarios, where it will be a full URL - "http://host:port/..."
+    var reminderUrlRegX = new RegExp("^(?:" + serverUrl + "|" + serverUrl.origin + "|" + serverUrl.pathname + ")(.*)");
+    var reminderUrlMatch = reminderUrlRegX.exec(absBundleUrl);
```
Comment (davidmsantos90) [inline impl/client/src/main/javascript/web/pentaho/i18n.js]: This two variables should be 'remainderUrlRegX' and 'remainderUrlMatch' respectively

**Comment by Advent51** [comment]
@davidmsantos90 please review

**Pull request merged**